saleor/payment/gateways/braintree/__init__.py

```diff
@@ -5,6 +5,7 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import pgettext_lazy
 
+from ...gateway_dataclasses import GatewayResponse
```

Comment:
I would think of some other name, I mean, we don't have to say in the filename that we use dataclasses dor that ;) What about `interface.py`? It would be a good name if we're going to extend it with some other common types in the future. But I'm open to suggestions.

Reply:
I was going to ask for a better name 👍  Thanks @maarcingebala